### DISCUSSION TITLE
Mobile App - UIUX

### DISCUSSION THREAD

**Discussion by Israelrex9** [keep-starknet-strange/joyboy#48] [Ideas]
I prepared a UI of how the app could look like, implementing the Status updates, like we have on already socials (instagram and warpcast) but I call these streaks in joyboy.

Here is a link to the Figma prototype, feel free to go around the UI. [Figma Prototype](https://www.figma.com/proto/NIJ68MV1AeI8y283woXiM0/JoyBoy?page-id=0%3A1&node-id=33-6304&viewport=687%2C228%2C0.24&t=hXCByjHbgHvL1XKi-1&scaling=min-zoom&starting-point-node-id=33%3A6304&show-proto-sidebar=1)

Also find attached a screen record of the App UIUX and flow

https://github.com/keep-starknet-strange/joyboy/assets/46507005/f35af8d2-c9b3-4f6d-b17e-c3c27b34aef1

**Comment by okhaimie-dev** [comment]
nice start bro! i will love to see the tipping flow. i.e. 

- ( user story: as a user, I will love to tip a connection / post maker for a valuable post they made. )
- ( user story: as a user, i will love to view my tip jar to see how much tips i got in the month of May. )

**Reply by Israelrex9** [reply]
oh yes, in my next iterations.

**Reply by Israelrex9** [reply]
would it be okay if any user can see post makers total tips per post?

**Comment by ayushtom** [comment]
Should we add Starknet purple instead of blue? Just thinking out loud 😂 Wdyt?

also few things which I noticed - 
- Should we add a bigger cta to navigate to login?
- I think search will need a navigation button in the feed page to look through users and post
- Can we also add error states for login flow so that we make all the flows complete for all edge cases.

**Reply by Israelrex9** [reply]
lol yeah, I can use Starknet purple

hmm, the first screen is a splash screen, it shouldn't have that button. It only stays for 3 seconds (i believe) before moving to the next screen for login and create account. 

Do you mean the search will be on the navigation tab or on the feed page?

Surely, I can include error states.

**Reply by ayushtom** [reply]
- I think i saw the arrow button below and i assumed that is not the splash screen. Is it intended?
- yea i think on the navigation bar of the feedpage i think it should be there, wdyt?

**Reply by Israelrex9** [reply]
yeah, the arrow, was for the prototype actually, you click on it to move to the next.
yeah, we can have the search on the navigation bar.

**Reply by Israelrex9** [reply]
![starknet-logo-light 1](https://github.com/keep-starknet-strange/joyboy/assets/46507005/c48e225c-3bc7-4828-b358-53046cde33be)

Starknet doesn't seem to have purple but blue @ayushtom
